modules/battenberg/1.1/config/default.yaml

```diff
@@ -10,18 +10,15 @@ lcr-modules:
 
         scratch_subdirectories: []
 
-        reference_path:
-            hg38: "__UPDATE__"
-            grch37: "__UPDATE__"
-
         options:
```

Comment:
This is a feature of the genome builds. If those are auto-downloaded, then it doesn't need to (and shouldn't be) in the configs. You still need to handle it properly so the script works.

Reply:
Already deleted and tested without it